sdk/include/opentelemetry/sdk/metrics/export/periodic_exporting_metric_reader_options.h

```diff
@@ -24,10 +28,11 @@ struct PeriodicExportingMetricReaderOptions
 {
   /* The time interval between two consecutive exports. */
   std::chrono::milliseconds export_interval_millis =
-      std::chrono::milliseconds(kExportIntervalMillis);
+      GetEnvDuration("OTEL_METRIC_EXPORT_INTERVAL", kExportIntervalMillis);
```

Comment:
Please move this initialization to the *.cc file.

It should not be in the header, we don't want to expose more symbols to the client application code that configures the sdk.

Please move the kExportIntervalMillis constant to *.cc as well.

Reply:
> However, I can't move the kExportIntervalMillis and kExportTimeOutMillis to the *.cc file since they are used in the periodic_exporting_metric_reader.cc in case the interval is inferior to the timeout.

Thanks for the details.
Ok to leave constants where they are then.